src/parser/monk/mistweaver/modules/features/MistweaverHealingEfficiencyTracker.tsx

```diff
@@ -38,7 +39,22 @@ class MistweaverHealingEfficiencyTracker extends HealingEfficiencyTracker {
     expelHarm: ExpelHarm,
   };
 
-  getCustomSpellStats(spellInfo, spellId) {
+  protected manaTracker!: ManaTracker;
+  protected abilityTracker!: AbilityTracker;
+  protected healingDone!: HealingDone;
+  protected damageDone!: DamageDone;
+  protected castEfficiency!: CastEfficiency;
+  protected abilities!: Abilities;
+  protected essenceFont!: EssenceFont;
+  protected essenceFontMastery!: EssenceFontMastery;
+  protected envelopingMists!: EnvelopingMists;
+  protected soothingMist!: SoothingMist;
+  protected renewingMist!: RenewingMist;
+  protected vivify!: Vivify;
+  protected refreshingJadeWind!: RefreshingJadeWind;
+  protected expelHarm!: ExpelHarm;
+
+  getCustomSpellStats(spellInfo: {spell: Spell; casts: number; healingHits: number; healingDone: number; overhealingDone: number; damageHits: number; damageDone: number; damageAbsorbed: number; manaSpent: number; manaGained: ManaTracker;}, spellId: number) {
```

Comment:
as said in discord, probably create an interface for spellInfo

Reply:
definitely; in the end it will be replaced by whoever converts HealingEfficiencyTracker, but even in the short term you have the object repeated like 6 times, it needs to be defined as a type alias or an interface

Reply:
I didn't know that you could create a interface in the child class and not have things heck up. I'll create one